src/loc/ContextualizedLocDetails.tsx

```diff
@@ -395,14 +395,16 @@ export default function ContextualizedLocDetails(props: Props) {
             <CertificateAndLimits
                 locId={ locId }
                 loc={ loc }
+                viewer={ props.viewer }
```

Comment:
Maybe we should move that "viewer" property to the LocContext? Can be done later.

Reply:
Good idea, but CommonContext seems a better candidate for that kind of info.